<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionWindow</class>
 <widget class="QDialog" name="OptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="navBar_options">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>550</height>
      </size>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(150, 150, 150, 255));  
border-right: 2px groove gray; 
}

QListWidget::item:hover 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(200, 200, 200, 255)); 
}

QListWidget::item:selected 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
}

QListWidget::item 
{ 
border-bottom: 2px groove gray; 
color: white;
padding: 10px;
}
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/options_gui.png</normaloff>:/images/options_gui.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Paths</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/options_folder.png</normaloff>:/images/options_folder.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Data</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/options_data.png</normaloff>:/images/options_data.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Network</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/options_network.png</normaloff>:/images/options_network.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>490</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>490</width>
       <height>550</height>
      </size>
     </property>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>511</y>
        <width>301</width>
        <height>32</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="txt_optionTitle">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>6</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stacked_optionPages">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>480</width>
        <height>430</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>430</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>430</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_general">
       <widget class="QComboBox" name="combo_option_language">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>185</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_theme">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>145</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Theme</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_language">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>145</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_option_autolanguage">
        <property name="geometry">
         <rect>
          <x>182</x>
          <y>30</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Automatic language detection</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_option_themes">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>185</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_option_hardwaredoubleclick">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_hwdoubledefault">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>461</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Default action when double-clicking on hardware module</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_search_default">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>461</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Hardware to open on B&amp;R homepage / search for in B&amp;R help</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_option_hwSearch">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>129</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Plugged</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Configured</string>
         </property>
        </item>
       </widget>
       <widget class="QCheckBox" name="check_option_splashScreen">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>216</y>
          <width>451</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Show splash screen at startup</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_option_autoCheckUpdates">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>461</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Check for updates on startup</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_paths">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>461</width>
          <height>203</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="text_options_websearch_prefix">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Websearch prefix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="label_options_websearch_prefix"/>
         </item>
         <item>
          <widget class="QLabel" name="text_options_websearch_suffix">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Websearch suffix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="label_options_websearch_suffix"/>
         </item>
         <item>
          <widget class="QLabel" name="text_options_serialsearch_prefix">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Serialsearch prefix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="label_options_serialsearch_prefix"/>
         </item>
         <item>
          <widget class="QLabel" name="text_options_serialsearch_suffix">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Serialsearch suffix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="label_options_serialsearch_suffix"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="text_options_waitforASHelp_ms">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>124</y>
          <width>169</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_options_helpexplorer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>461</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="label_options_timetowaitforhelp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>126</y>
          <width>57</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_waitforASHelp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>106</y>
          <width>451</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Time to wait for AS Help to open</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_brhelpexplorer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Path to Br.Help.Explorer.exe</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_options_browseBRhelp">
        <property name="geometry">
         <rect>
          <x>396</x>
          <y>52</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_options_helplanguage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>57</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Help language to open</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_option_helplanguage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>185</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_data">
       <widget class="QCheckBox" name="check_option_keepFiles">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>361</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Keep temporary downloaded files</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_option_downlaodData">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>34</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Download data objects (NCT, .br-Modules ...)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="text_options_updateserver">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>459</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Updateserver</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="label_options_updateserver">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>459</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>https://github.com/bee-eater/SystemDumpViewer/releases/latest</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_option_ignoresslerrors">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>451</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ignore SSL errors</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="graphic_option_bg">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>501</width>
        <height>601</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>501</width>
        <height>601</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>501</width>
        <height>601</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGraphicsView {
	background : transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <zorder>graphic_option_bg</zorder>
     <zorder>stacked_optionPages</zorder>
     <zorder>buttonBox</zorder>
     <zorder>txt_optionTitle</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
